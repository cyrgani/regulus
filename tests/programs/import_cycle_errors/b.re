def(from_b, print("bb")),

import(a),
print(from_a()),
