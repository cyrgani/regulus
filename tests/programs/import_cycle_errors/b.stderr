
ImportError: cyclic import of `a` at path `programs/import_cycle_errors/a.re` detected
at programs/import_cycle_errors/b.re:3:7